src/app/modules/users/pages/users.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-users',
+  templateUrl: './users.component.html',
+  styleUrls: ['./users.component.css']
+})
+export class UsersComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
what for? this component should not implement anything

Reply:
These are functions that come by default when generating a component in angular, but now I remove them